docs/4.3/enterprise/workflow/ssh_approval_slack.md

```diff
@@ -0,0 +1,223 @@
+root-cas = "/var/lib/teleport/plugins/slackbot/auth.cas"   # Teleport cluster CA certs
+
+[slack]
+token = "api-token"       # Slack Bot OAuth token
+secret = "signing-secret-value"   # Slack API Signing Secret
+channel = "channel-name"  # Slack Channel name to post requests to
+
+[http]
+listen = ":8081"          # Slack interaction callback listener port
+# https-key-file = "/var/lib/teleport/plugins/slackbot/server.key"  # TLS private key
+# https-cert-file = "/var/lib/teleport/plugins/slackbot/server.crt" # TLS certificate
+
+[log]
+output = "stderr" # Logger output. Could be "stdout", "stderr" or "/var/lib/teleport/slackbot.log"
+severity = "INFO" # Logger severity. Could be "INFO", "ERROR", "DEBUG" or "WARN".
+```
+
+## Test Run 
+
+Assuming that Teleport is running, and you've created the Slack app, the plugin config, and provided all the certificates — you can now run the plugin and test the workflow!
+
+`teleport-slackbot start`
+
+If everything works fine, the log output should look like this:
+
+```bash
+vm0:~/slackbot sudo ./teleport=slackbot start
+INFO   Starting Teleport Access Slackbot 0.1.0: slackbot/main.go:224
+INFO   Starting a request watcher... slackbot/main.go:330
+INFO   Starting insecure HTTP server on 0.0.0.0:8081 utils/http.go:64
+INFO   Watcher connected slackbot/main.go:298
+```
+
+### Testing the approval workflow
+
+You can create a test permissions request with `tctl` and check if the plugin works as expected like this: 
+
+#### Create a test permissions request
+
+```bash
+tctl request create USERNAME --roles=TARGET_ROLE
+```
+
+#### Check that you see a request message on Slack 
+
+It should look like this: %image%
+
+#### Approve or deny the request on Slack
+
+The messages should automatically get updated to reflect the action you just clicked. 
+
+You can also check the request status with `tctl`: 
+
+```bash
+tctl request ls
+```
+
+### TSH User Login and Request Admin Role. 
+
+You can also test the full workflow from the user's perspective using `tsh`: 
+
+```bash
+➜ tsh login --request-roles=REQUESTED_ROLE
+Seeking request approval... (id: 8f77d2d1-2bbf-4031-a300-58926237a807)
+```
+
+You should now see a new request in Teleport, and a message about the request on Slack. You can approve or deny it and `tsh`should login successfully or error out right after you click an action button on Slack.
+
+
+### Setup with SystemD
+In production, we recommend starting teleport plugin daemon via an init system like systemd . Here's the recommended Teleport Plugin service unit file for systemd: 
+
+```
+[Unit]
+Description=Teleport Plugin
+After=network.target
+
+[Service]
+Type=simple
+Restart=on-failure
+ExecStart=/usr/local/bin/teleport-slackbot start --config=/etc/teleport-slackbot.toml --pid-file=/var/run/teleport-slackbot.pid
+ExecReload=/bin/kill -HUP $MAINPID
+PIDFile=/var/run/teleport-slackbot.pid
+
+[Install]
+WantedBy=multi-user.target
+```
```

Comment:
We should provide these `systemd` units for every plugin.

Reply:
I've added these